apps/web/screens/OfficialJournalOfIceland/OJOISearch.tsx

```diff
@@ -0,0 +1,562 @@
+              options={[
+                { ...emptyOption('Allir flokkar') },
+                ...categoriesOptions,
+              ]}
+              isClearable
+              isSearchable
+              value={findValueOption(
+                categoriesOptions,
+                searchState.malaflokkur,
+              )}
+              onChange={(v) => updateSearchState('malaflokkur', v?.value ?? '')}
+            />
+
+            <DatePicker
+              size="xs"
+              locale="is"
+              name="dagsFra"
+              label={o('dateFromLabel', 'Dags. frá')}
+              placeholderText={o(
+                'dateFromPlaceholder',
+                'Veldu upphafsdagsetningu',
+              )}
+              selected={
+                searchState.dagsFra ? new Date(searchState.dagsFra) : undefined
+              }
+              minDate={new Date('1950-01-01')}
+              maxDate={
+                searchState.dagsTil ? new Date(searchState.dagsTil) : undefined
+              }
+              handleChange={(date) =>
+                updateSearchState(
+                  'dagsFra',
+                  date ? date.toISOString().slice(0, 10) : '',
+                )
+              }
+            />
+
+            <DatePicker
+              size="xs"
+              locale="is"
+              name="dagsTil"
+              label={o('dateToLabel', 'Dags. til')}
+              placeholderText={o('dateToPlaceholder', 'Veldu lokadagsetningu')}
+              selected={
+                searchState.dagsTil ? new Date(searchState.dagsTil) : undefined
+              }
+              minDate={
+                searchState.dagsFra ? new Date(searchState.dagsFra) : undefined
+              }
+              maxDate={new Date()}
+              handleChange={(date) =>
+                updateSearchState(
+                  'dagsTil',
+                  date ? date.toISOString().slice(0, 10) : '',
+                )
+              }
+            />
+
+            <Select
+              name="stofnun"
+              label={o('institutionLabel', 'Stofnun')}
+              size="xs"
+              placeholder={o('institutionPlaceholder', 'Veldu stofnun')}
+              options={[
+                { ...emptyOption('Allir stofnanir') },
+                ...institutionsOptions,
+              ]}
+              isClearable
+              isSearchable
+              value={findValueOption(institutionsOptions, searchState.stofnun)}
+              onChange={(v) => updateSearchState('stofnun', v?.value ?? '')}
+            />
+          </Stack>
+        </Box>
+      }
+      breadcrumbItems={breadcrumbItems}
+    >
+      {adverts?.length ? (
+        <Stack space={3}>
+          <Button
+            onClick={() => setListView(!listView)}
+            size="small"
+            iconType="outline"
+            icon={listView ? 'copy' : 'menu'}
+            variant="utility"
+          >
+            {listView ? 'Sýna sem spjöld' : 'Sýna sem listi'}
```

Comment:
I wonder if "Sýna sem listi" is correct Icelandic grammar, should it rather be "Sýna sem lista". I'm not sure though 😅